### DISCUSSION TITLE
Updating Playlist Directory with New Tracks

### DISCUSSION THREAD

**Discussion by randomscumbag** [strawberrymusicplayer/strawberry#1442] [General]
I am not sure if this is possible already or not, I just haven't figured it out.  Sorry if my wording is bad.  I currently have saved playlists which I access often.  However I add new tracks to that collection throughout the month.  I usually clear and then add the folder back so all the tracks are in there.  I can also add the folder and remove duplicates.  I was wondering if there was a way to Update the Playlist to add the new files or remove the files from the playlist if they were removed.  I know theres Remove Unavailable which I use, but pretty much just an Update Playlist? I guess is what I am looking for.  Again not sure if this already in the Program and im simply just missing the steps.  Either way Thank you.

**Comment by jonaski** [comment]
Currently not possible, suggest it on: https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
